bin/build.js

```diff
@@ -3,21 +3,46 @@
 const fs = require('fs');
 
 let original = '';
+let version = '';
 
+// Read version from the normalize.css package and write it in our package.json
+const normalizePackage = fs.createReadStream('./node_modules/normalize.css/package.json', 'utf8');
```

Comment:
Since this is in build time, and the files aren't big, why not make your life and future maintenance easier and use sync versions of fs methods?